lensless/unrolled_admm.py

```diff
@@ -0,0 +1,248 @@
+# #############################################################################
+# unrolled_admm.py
+# =================
+# Authors :
+# Yohann PERRON [yohann.perron@gmail.com]
+# #############################################################################
+
+import numpy as np
+from lensless.trainable_recon import TrainableReconstructionAlgorithm
+from lensless.admm import soft_thresh, finite_diff, finite_diff_adj, finite_diff_gram
+from scipy import fft
+
+try:
+    import torch
+
+    torch_available = True
+except ImportError:
+    torch_available = False
+
+
+class UnrolledADMM(TrainableReconstructionAlgorithm):
+    """
+    Object for applying ADMM (Alternating Direction Method of Multipliers) with
```

Comment:
Update docstring that this is unrolled version of ADMM and remove links to paper/slides. Can directly link to other docstring with something like
```
Object for applying unrolled version of :py:class:`~lensless.ADMM`.
```